File: MQTT_JavaScript/subApp.js
var mqtt = require('mqtt')      // npm install mqtt
var client  = mqtt.connect('mqtt://localhost')

var topic = 'LINTANGtopic123'
client.on('connect', function () {
    client.subscribe(topic)
})

client.on('message', function (topic, message) {
    context = message.toString();
    console.log(context)
})

File: MQTT_Python/subApp.py
# pip install paho-mqtt
import paho.mqtt.client as mqtt

def on_connect(client, userdata, flags, rc):
    print("Connected with result code "+str(rc))
    client.subscribe("LINTANGtopic/test")

def on_message(client, userdata, msg):
    print(msg.payload.decode())

client = mqtt.Client()
client.connect("localhost",1883,60)

while True:
    client.on_connect = on_connect
    client.on_message = on_message
    client.loop_forever()